### ISSUE TITLE
Text input not working on jquery modal dialog

### ISSUE THREAD

**Issue by bmsmg** [claviska/jquery-minicolors#82]
I have found an issue when trying to use minicolors on a jquery modal dialog.  I cannot get the text input to work on the modal.  Whenever I click on the input to key in a hex color, it appears as if the textbox is disabled.  I have debugged in Chrome, and it appears the input's focus event is firing properly.  I'm using jquery 1.83 and jquery-ui 1.92  I have created a simple example to demonstrate the issue (apologies for the lack of formatting, but you should be able to copy/paste into an html doc that includes the jquery.minicolors script and image files):

<pre>
<code>
&lt;!DOCTYPE html&gt;
&lt;html&gt;
&lt;head&gt;
    &lt;title&gt;Jquery MiniColors Not Working On Jquery Dialog Modal&lt;/title&gt;
    &lt;link href="jquery.miniColors.css" rel="stylesheet" /&gt;
    &lt;style&gt;
        .dialog { border: 2px solid green; }
        .modal-dialog { border: 2px solid red; }
    &lt;/style&gt;
    &lt;script src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"&gt;&lt;/script&gt;
    &lt;script src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.9.2/jquery-ui.min.js"&gt;&lt;/script&gt;
    &lt;script src="jquery.minicolors.js"&gt;&lt;/script&gt;
    &lt;script&gt;
        $( document ).ready( function () {
            $( '.colorpicker' ).minicolors();

            $( '#dlg' ).click( function () {
                $( '&lt;div class="dialog"&gt;&lt;input type="text" class="colorpicker" /&gt;&lt;/div&gt;' ).dialog( {
                    autoOpen: true,
                    open: function ( e, ui ) {
                        $( e.target ).find( '.colorpicker' ).minicolors();
                    }
                } );
            } );
            
            $( '#mdlg' ).click( function () {
                $( '&lt;div class="modal-dialog"&gt;&lt;input type="text" class="colorpicker" /&gt;&lt;/div&gt;' ).dialog( {
                    autoOpen: true,
                    modal: true,
                    open: function ( e, ui ) {
                        $( e.target ).find( '.colorpicker' ).minicolors();
                    }
                } );
            } );
        } )
&lt;/script&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;label for="picker"&gt;Keying in color codes works fine here...&lt;/label&gt;
    &lt;input id="picker" type="text" class="colorpicker" /&gt;&lt;br/&gt;
    &lt;label for="dlg"&gt;It also works fine on a non-modal dialog...&lt;/label&gt;
    &lt;button id="dlg"&gt;Launch Non-Modal Dialog&lt;/button&gt;&lt;br/&gt;
    &lt;label for="mdlg"&gt;But the text input doesn't work when on a modal dialog...&lt;/label&gt;
    &lt;button id="mdlg"&gt;Launch Modal Dialog&lt;/button&gt;
&lt;/body&gt;
&lt;/html&gt;
</code>
</pre>

Do you have any ideas?

**Comment by kayseripanel** [comment]
You can  use modal:false .

**Comment by Vue-class-components-pet** [comment]
@claviska https://stroller-apache-camel.github.io/thickness/
